astropy/coordinates/tests/test_sites.py

```diff
@@ -86,6 +86,38 @@ def test_EarthLocation_basic():
         EarthLocation.of_site("nonexistent")
 
 
+@pytest.mark.parametrize(
+    "class_method,args",
+    [(EarthLocation.get_site_names, []), (EarthLocation.of_site, ["greenwich"])],
+)
+def test_Earthlocation_refresh_cache_is_mandatory_kwarg(args, class_method):
```

Comment:
It is confusing to have the parametrize order to be opposite of actual args order.
```suggestion
def test_Earthlocation_refresh_cache_is_mandatory_kwarg(class_method, args):
```

Reply:
I ordered the arguments alphabetically, but the alphabetical ordering is not important here so it might not have been helpful.

Reply:
I will leave issues with the unit test to @eerovaher to resolve.